test/draw/gpu/BlitTest.ooc

```diff
@@ -41,6 +50,22 @@ BlitTest: class extends Fixture {
 			this testOffset(this sourceImage, correctImage, IntVector2D new(3, 2))
 			correctImage free()
 		})
+		this add("Unaligned", func {
+			correctImage := RasterMonochrome fromAscii(
+				"< *X>
+				<               >
+				<               >
+				<   XXXXXXXX    >
+				<   X*    *X    >
+				<   X*    *X    >
+				<   X*   XXX    >
+				<   X****XXX    >
+				<   XXXXXXXX    >
+				<               >"
+			)
+			this testOffset(this unalignedSourceImage, correctImage, IntVector2D new(3, 2))
+			correctImage free()
+		})
```

Comment:
Both input and output is unaligned so that breaking the automatic padding will be caught.